pr-data.csv

```diff
@@ -2625,20 +2625,20 @@ https://github.com/apache/opennlp,8209cd308ee6ff7e61ec0346f1799a32541ed80f,openn
 https://github.com/apache/opennlp,8209cd308ee6ff7e61ec0346f1799a32541ed80f,opennlp-tools,opennlp.tools.languagemodel.NgramLanguageModelTest.testTrigram,ID,Accepted,https://github.com/apache/opennlp/pull/387,
 https://github.com/apache/opennlp,8209cd308ee6ff7e61ec0346f1799a32541ed80f,opennlp-tools,opennlp.tools.ml.naivebayes.NaiveBayesSerializedCorrectnessTest.testPlainTextModel,ID,Accepted,https://github.com/apache/opennlp/pull/387,
 https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-impl,org.apache.webbeans.test.annotation.binding.BeanCacheKeyUnitTest.testEquals2Annotations,ID,Rejected,https://github.com/apache/openwebbeans/pull/53,https://github.com/TestingResearchIllinois/idoft/issues/561
-https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-impl,org.apache.webbeans.test.annotation.binding.BeanCacheKeyUnitTest.testEquals2AnnotationsUnorderedParam,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/561
-https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-impl,org.apache.webbeans.test.proxy.InterceptorProxySerializationTest.testSerializableEvenIfAutoIntercepted,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/561
+https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-impl,org.apache.webbeans.test.annotation.binding.BeanCacheKeyUnitTest.testEquals2AnnotationsUnorderedParam,ID,DeveloperWontFix,https://github.com/apache/openwebbeans/pull/53,https://github.com/TestingResearchIllinois/idoft/issues/561 [test fails only in FULL mode not in ONE mode]
```

Comment:
I might have missed some context, but this PR is not merged (the PR came from another Darko's student). If the test has been fixed in the latest commit, please fine the commit that fixed the test.